api/src/main/java/org/apache/iceberg/Table.java

```diff
@@ -138,6 +138,21 @@ default String name() {
    */
   List<HistoryEntry> history();
 
+  /**
+   * Get the snapshot references of this table.
```

Comment:
worth pointing out that there is always a snapshot ref for the main branch. If there is no snapshot in the table, main branch points to -1